stories/module-analytics-settings.stories.js

```diff
@@ -101,6 +104,61 @@ function Settings( props ) {
 	);
 }
 
+function makeGtmPropertyStory( { permission, useExistingTag = false } ) {
```

Comment:
I could be wrong but I think this may have been a mistake with the IB as I don't think this issue is intended to change the Settings components. cc @felixarntz